scipy/sparse/linalg/_dsolve/tests/test_linsolve.py

```diff
@@ -732,7 +773,7 @@ def test_zero_diagonal(self):
         rng = np.random.default_rng(43876432987)
         A = rng.standard_normal((n, n))
         b = np.arange(n)
-        A = scipy.sparse.tril(A, k=0, format='csr')
+        A = scipy.sparse.tril(A, k=0, format='csc')
```

Comment:
Change from CSR to CSC

Reply:
Thanks, made a parametric test with both formats in [e97868d](https://github.com/scipy/scipy/pull/17924/commits/e97868dfca28071066e0a38b3e9e41849eb4d713)